src/modelfree/common/utils.py

```diff
@@ -43,6 +46,9 @@ def save(self, save_path):
     def load(self):
         raise NotImplementedError()
 
+    def _get_pretrain_placeholders(self):
```

Comment:
Why do we need this? Not part of the standard `BaseRLModel` API AFAIK.

Reply:
It is in the `gail-test` branch which has the other gail-related fixes I've been using

Reply:
OK, fair enough, I see it got added in one of the recent PRs for Stable Baselines